### PR TITLE
Add documentation for using image_variable in prompt templates

### PR DESCRIPTION
**Pull request by abubakarsohail**
Introduce documentation for the `image_variable` attribute, detailing its usage in prompt templates to dynamically insert images. Include an example template demonstrating how to implement this feature.

### CODE REVIEW

File: running-requests/prompt-blueprints.mdx [hunk 1]
```diff
@@ -132,3 +132,48 @@ response = pl_client.run(
 
 print(response)
 ```
+
+
+## Using Image Variable
+
+The `image_variable` attribute allows you to dynamically insert a list of images into your prompt templates. This attribute is part of the `ImageContent` structure and can be used to include images based on a specified variable name.
```
Comment (jzone3) [inline running-requests/prompt-blueprints.mdx:139]: into your prompt templates
->
into prompt template messages

Comment (jzone3) [inline running-requests/prompt-blueprints.mdx:139]: What is `ImageContent`?

File: running-requests/prompt-blueprints.mdx [hunk 2]
```diff
@@ -132,3 +132,48 @@ response = pl_client.run(
...
+
+### Structure
+
+The `image_variable` is nested within the message content.
+
+```json
+{
+    “role”: “user”,
+    “content”: [
+        {
```
Comment (jzone3) [inline running-requests/prompt-blueprints.mdx:149]: This is confusing.. are we adding the image or the the image variable. What is image url?

File: running-requests/prompt-blueprints.mdx [hunk 3]
```diff
@@ -132,3 +132,48 @@ response = pl_client.run(
...
+            “image_variable”: “image_variables”
+        }
+    ]
+}
+```
+
+### Example Usage
+
+When defining a prompt template, you can specify an `image_variable` to dynamically include images in your messages.
+<img src="/images/image-variable.mp4"/>
```
Comment (jzone3) [inline running-requests/prompt-blueprints.mdx:161]: use GIF not mp4

LMK I can get you Cleanshot X that's what I use

File: running-requests/prompt-blueprints.mdx (hunk 1 above)
Comment (abubakarsohail) [reply running-requests/prompt-blueprints.mdx:139]: Updated

Comment (abubakarsohail) [reply running-requests/prompt-blueprints.mdx]: We have two types of `content` inside a message body
1. `TextContent` which is supposed to contain text
2. `ImageContent` which is supposed to contain image content

File: running-requests/prompt-blueprints.mdx (hunk 2 above)
Comment (abubakarsohail) [reply running-requests/prompt-blueprints.mdx:149]: `image_variable` exists inside `ImageContent`. `image_url` can be used to reference images from a URL

File: running-requests/prompt-blueprints.mdx (hunk 3 above)
Comment (abubakarsohail) [reply running-requests/prompt-blueprints.mdx:161]: Yes, please. I couldn't find an easy to do it natively on macOS.

**Pull request merged**